docs/gl_objects/issues.rst

```diff
@@ -225,3 +225,51 @@ Link issue ``i1`` to issue ``i2``::
 Delete a link::
 
     i1.links.delete(issue_link_id)
+
+Issues statistics
+=========================
+
+Reference
+---------
+
+* v4 API:
+
+  + :class:`gitlab.v4.objects.IssuesStatistics`
+  + :class:`gitlab.v4.objects.IssuesStatisticsManager`
+  + :attr:`gitlab.issuesstatistics`
+  + :class:`gitlab.v4.objects.GroupIssuesStatistics`
+  + :class:`gitlab.v4.objects.GroupIssuesStatisticsManager`
+  + :attr:`gitlab.v4.objects.Group.issuesstatistics`
```

Comment:
Is it possible to do: `gitlab.v4.objects.Group.issues_statistics` ?

Changing `issuesstatistics` to `issue_statistics`